addons/web/static/src/legacy/js/views/res_partner_view_dialogs.js

```diff
@@ -0,0 +1,44 @@
+/** @odoo-module alias=web.ResPartnerFormViewDialog **/
+
+import { _t } from 'web.core';
+import Dialog from 'web.Dialog';
+import FormViewDialog from 'web.view_dialogs';
```

Comment:
just to save some characters
```suggestion
import {FormViewDialog} from 'web.view_dialogs';
```